torchvision/ops/feature_pyramid_network.py

```diff
@@ -80,19 +107,17 @@ def forward(self, x):
         names = list(x.keys())
         x = list(x.values())
 
-        last_inner = self.inner_blocks[-1](x[-1])
+        last_inner = self.last_inner_block(x[-1])
         results = []
-        results.append(self.layer_blocks[-1](last_inner))
-        for feature, inner_block, layer_block in zip(
-            x[:-1][::-1], self.inner_blocks[:-1][::-1], self.layer_blocks[:-1][::-1]
-        ):
-            if not inner_block:
```

Comment:
I think this part of the implementation is missing.
Before, it was possible to pass empty blocks so that we do not compute anything on some layers.
IIRC this is currently not used in the models in torchvision, but was used in another model (RetinaNet), which re-uses a lot of the common implementation.

Do you think it would be possible to use the implementation from before, or keep this `continue`?

Reply:
Unfortunately this isn't currently possible because modules are not first class values, so we can only desugar them in a simple for loop (without `zip`.). If that's a blocker or it makes the code unreadable we can also just not merge this and wait for that to be supported.

Reply:
let me check on Monday if this would be a blocker or not